resources/app/controllers/refinery/admin/resources_controller.rb

```diff
@@ -44,20 +44,24 @@ def create
 
       def insert
         self.new if @resource.nil?
-
-        @url_override = refinery.admin_resources_path(request.query_parameters.merge(:insert => true))
+        @url_override = refinery.admin_resources_path(request.query_parameters.merge(insert: true))
 
         if params[:conditions].present?
           extra_condition = params[:conditions].split(',')
 
           extra_condition[1] = true if extra_condition[1] == "true"
           extra_condition[1] = false if extra_condition[1] == "false"
           extra_condition[1] = nil if extra_condition[1] == "nil"
-          paginate_resources({extra_condition[0] => extra_condition[1]})
-        else
-          paginate_resources
         end
-        render 'insert'
+
+        find_all_resources(({extra_condition[0] => extra_condition[1]} if extra_condition.present?))
+        search_all_resources if searching?
+
+        paginate_resources
+
+        @resource_area_selected = from_dialog?
+
+        render @callback.presence || 'insert'
```

Comment:
Hi @parndt ! I don't find how i could render the template by context 'insert' or in '/refinery/admin/pages_dialogs/link_to' template.

Using @callback is not the good way, could you help me to find?

Reply:
oops you accidentally mentioned the `@callback` user 😉

Reply:
Oops! i will not expect a callback from this user 😆

Reply:
Any idea @parndt ?